tvb/datatypes/projections.py

```diff
@@ -78,7 +124,17 @@ class ProjectionRegionEEG(framework.ProjectionRegionEEGFramework,
         
     
     """
-    pass
+    @staticmethod
+    def from_file(source_file='projection_EEG_region.npy', instance=None):
```

Comment:
but I allow both methods: if a mat file is provided, they we assume it's a bst file, otherwise if npy, we assume you did it yourself


On Sun, Jun 7, 2015, at 02:15 PM, marmaduke woodman wrote:
> In tvb/datatypes/projections.py[1]:


> > @@ -78,7 +124,17 @@ class
> > ProjectionRegionEEG(framework.ProjectionRegionEEGFramework,
>
>
>      """
> -    pass
> +    @staticmethod
> +    def from_file(source_file='projection_EEG_region.npy',
>      instance=None): reading from bst .mat means not needing an
>      intermediate step to generate the npy file which is slightly more
>      trustworthy, but we can trust you this time ;)


> — Reply to this email directly or view it on GitHub[2].

>



Links:

  1. https://github.com/the-virtual-brain/tvb-library/pull/52#discussion_r31876394
  2. https://github.com/the-virtual-brain/tvb-library/pull/52/files#r31876394